src/main/groovy/edu/oregonstate/mist/permissionsapi/core/Permissions.groovy

```diff
@@ -0,0 +1,16 @@
+package edu.oregonstate.mist.permissionsapi.core
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+
+class Permissions {
+    @JsonIgnore
+    String osuID
+
+    String onid
```

Comment:
I know we already went through the swagger design, but I have a small change to request. Could we use the key `username` instead of `onid`? This will make it more aligned with our other APIs: https://github.com/osu-mist/persons-api/blob/a0618ea976b3df87606ab3dc70a3e8fb52f0873b/swagger.yaml#L232-L234